src/reducers/uiReducer.js

```diff
@@ -52,17 +52,11 @@ const initialStateNavigation = [
     isSelected: false
   },
   /*{
```

Comment:
hmmm, not sure. there are some "hidden" routes that we use for deep linking to both the full details PAGE and the page that shows all of a particular PR. Not sure which if any of these is "inspect".

Also, will you need to uncomment this to have Browse show up in the UI?